pages/dkp/kommander/2.1/install/small-environment/index.md

```diff
@@ -0,0 +1,92 @@
+
+## Minimal Kommander installation
+
+The `yaml` file that is used to install a minimal configuration of Kommander contains the bare minimum setup that allows you to deploy applications, access the DKP UI, create and attach clusters. It does **NOT** include applications for cost monitoring, logging, alerting, object storage, etc.
+
+In this `yaml` file you can find commented lines, that correspond to all platform applications which would be included in normal Kommander setup. Since these lines are commented, they are not taken into account during installation. If you want to test an additional platform application, you can enable it individually to be installed by uncommenting the corresponding line in the `yaml` file.
+
+<p class="message--important"><strong>IMPORTANT: </strong>Some applications depend on other applications to work properly. Refer to the <a href="../../workspaces/applications/platform-applications/platform-application-dependencies/">dependencies documentation</a> to find out which other applications you need to enable to test the target application.</p>
+
+For example, if you want to test Grafana to allow monitoring, remove the pound sign preceding `grafana-logging: null`. Since Grafana depends on Loki, you also have to remove the pound preceding `grafana-loki: null` and `minio-operator: null`. Note that depending on the size of the Kubernetes cluster, enabling several platform applications could exhaust your cluster’s resources.
+
+1.  Edit your `cluster.yaml` file to match the following example:
+
+    apiVersion: config.kommander.mesosphere.io/v1alpha1
+    kind: Installation
+    apps:
+    #centralized-grafana: null
+    #centralized-kubecost: null
+    dex: null
+    dex-k8s-authenticator: null
+    #fluent-bit: null
+     #gitea is only supported from DKP 2.2 and later
+     #gitea: null
+    #gatekeeper: null
+    #grafana-logging: null
+    #grafana-loki: null
+    #karma: null
+    #karma-traefik: null
+    kommander: null
+    #kube-prometheus-stack: null
+    #kubecost: null
+    #kubecost-thanos-traefik: null
+    kubefed: null
+    #kubernetes-dashboard: null
+    #kubetunnel: null
+     #kube-oidc-proxy was not included in the generated --init
+    kube-oidc-proxy: null
+    #logging-operator: null
+    #minio-operator: null
+    #metallb:
+        #values: |
+        #configInline:
+            #address-pools:
+            #- name: default
+            #protocol: layer2
+            #addresses:
+            #- 192.168.251.1-192.168.251.254
+    #prometheus-adapter: null
+    #prometheus-thanos-traefik: null
+    reloader: null
+    #thanos: null
+    traefik: null
+    traefik-forward-auth-mgmt: null
+    #velero: null
+    ageEncryptionSecretName: sops-age
+    clusterHostname: ""
+    appManagementImageTag: v2.1.1
+    appManagementImageRepository: mesosphere/kommander2-appmanagement
+    kommanderChartsVersion: v2.1.1
+
+1.  Create your Kommander cluster using the following command:
```

Comment:
I'm not sure if "create your Kommander cluster" is the right term as the command that follows doesn't create the cluster but only installs Kommander on an **existing** cluster.